modules/localization/msf/local_pyramid_map/ndt_map/ndt_map_matrix_handler.cc

```diff
@@ -0,0 +1,63 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+#include "modules/localization/msf/local_map/ndt_map/ndt_map_matrix_handler.h"
+
+namespace apollo {
+namespace localization {
+namespace msf {
+// =================PyramidMapMatrixHandlerSelector=================
+NdtMapMatrixHandlerSelector::NdtMapMatrixHandlerSelector() {}
+
+NdtMapMatrixHandlerSelector::~NdtMapMatrixHandlerSelector() {}
+
+BaseMapMatrixHandler* NdtMapMatrixHandlerSelector::AllocNdtMapMatrixHandler() {
+  return new NdtMapMatrixHandler();
```

Comment:
Use safe pointers

Reply:
The caller will use safe pointers to pack it.